### DISCUSSION TITLE
Freenginx: the fork of Nginx

### DISCUSSION THREAD

**Discussion by vitormattos** [nginx-proxy/nginx-proxy#2416] [Ideas]
A big developer of Nginx, Maxim Dounin, created a fork of Nginx and created the follow at Nginx mail list:

https://mailman.nginx.org/pipermail/nginx-devel/2024-February/K5IC6VYO2PB7N4HRP2FUQIBIBCGP4WAU.html

Will be very nice to have a version of nginx-proxy based on Freenginx.

**Comment by buchdag** [comment]
While I agree with the "nice to have", the realistic answer is that this won't happen unless there is a multi architecture Docker image of freenginx available and actively maintained by the freenginx dev / community, plus people able to handle the maintenance of the freenginx version here.

Maintaining our own freenginx Docker image and/or our own CI to build freenginx ourselves for multiple architectures is way too much work for the ressources at hand.

**Comment by flexiondotorg** [comment]
EDIT - Sorry, it appears I pasted a reply into the wrong issue.
